src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -37,12 +26,10 @@ export const StakingOpportunities = ({ assetId: caip19 }: StakingOpportunitiesPr
       <Card.Body pt={0}>
         <Stack spacing={2} mt={2} mx={-4}>
           {validators.map(validator => (
-            <StakingOpportunitiesRow name={validator.name} />
+            <StakingOpportunitiesRow name={validator.name} key={validator.id} />
           ))}
         </Stack>
       </Card.Body>
     </Card>
   )
-
-  return useRenderForChains([ChainTypes.Cosmos], cosmosStaking)
```

Comment:
I like this. Using the `CosmosInvestor` feature flag for conditionally rendering is way cleaner and a better development experience than always having to modify this.